administrator/language/en-GB/plg_fields_note.ini

```
@@ -0,0 +1,15 @@
+; Joomla! Project
+; (C) 2025 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_FIELDS_NOTE="Fields - Note"
+PLG_FIELDS_NOTE_LABEL="Note (%s)"
+PLG_FIELDS_NOTE_PARAMS_CLASS_DESC="The class used on the div of the note"
+PLG_FIELDS_NOTE_PARAMS_CLASS_LABEL="Note Class"
+PLG_FIELDS_NOTE_PARAMS_CLOSE_DESC="This field controls the display of an "x" to close the note. It takes a value of 'true' (for alerts) or the value for the data-dismiss of the Bootstrap close icon."
+PLG_FIELDS_NOTE_PARAMS_CLOSE_LABEL="Show close button"
```

```suggestion
PLG_FIELDS_NOTE_PARAMS_CLOSE_LABEL="Show Close Button"
```

our UI styleguide says that labelsshould be capitalised